test/shell_test.exp

```diff
@@ -423,6 +423,8 @@ njs_run {"-c" "console.log(process.pid)"} "\\d+"
 
 njs_run {"-c" "console.log(process.ppid)"} "\\d+"
 
+njs_run {"-c" "console.log(process.kill(process.pid, 0))"} "true"
```

Comment:
The tests runs `kill(pid, 0)` which just checks that the process exists. It doesn't kill anything.